lib/widget/webview/native/webviewstate.dart

```diff
@@ -86,33 +76,63 @@ class WebViewState extends WidgetState<EnsembleWebView> {
     );
   }
 
+  Stream<dynamic> onSetCookie() async* {
+    var cookieList = widget.controller.cookies;
+    for (var i in cookieList) {
```

Comment:
use better variable names. what is 'i'. Just call it cookie. Make it readable please, someone has to maintain it as well